openmdao/core/group.py

```diff
@@ -176,9 +176,15 @@ def connect(self, source, targets, src_indices=None):
             If specified, connect the specified entries of the full
             distributed source value to the target.
         """
+
         if isinstance(targets, str):
             targets = (targets,)
 
+        if isinstance(src_indices, str):
+            suggestion = (*targets, src_indices)
```

Comment:
apparently python 2 doesn't like using *targets if it's not being passed as a function arg. This line gives me a syntax error.